tests/test_pptx_worker/test_visualization/test_storage/test_gcs_client.py

```
@@ -0,0 +1,199 @@
+
+        assert key == "jobs/job-42/current.pptx"
+        mock_bucket.blob.assert_called_once_with("jobs/job-42/current.pptx")
+        mock_blob.upload_from_filename.assert_called_once_with(
+            str(src),
+            content_type="application/vnd.openxmlformats-officedocument.presentationml.presentation",
+        )
+
+
+class TestUploadPdf:
+    def test_returns_canonical_key(self, mock_gcs, tmp_path):
+        client, mock_bucket, mock_blob, _ = mock_gcs
```

Ruff (0.15.13)

[warning] Unpacked variable `mock_bucket` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)